tools/sz_repo_cli/lib/src/commands/src/pub_get_command.dart

```diff
@@ -48,13 +41,43 @@ class PubGetCommand extends Command {
 
   @override
   Future<void> run() async {
-    final includeFlutterApp = argResults[includeFlutterAppFlagName] ?? true;
+    isVerbose = argResults['verbose'] ?? false;
 
-    final results = _useCase.getPackages(
-      includeFlutterApp: includeFlutterApp,
-      packageTimeout: argResults.packageTimeoutDuration,
+    final _max = argResults[maxConcurrentPackagesOptionName];
+    final maxNumberOfPackagesBeingProcessedConcurrently = _max != null
+        ? int.tryParse(argResults[maxConcurrentPackagesOptionName])
+        // null wird nachher als "keine Begrenzung" gehandhabt.
```

Comment:
Can we new code only write in English since the app open source?

Reply:
I just copy-pasted it from the other instances. Fixed it here, for the other instances I'll do it in the future in another PR.